### PR TITLE
Update associating-user-accounts.md

### PR DESCRIPTION
**Pull request by Sahilgandas1**
fix for issue https://github.com/wso2/product-is/issues/11184

### CODE REVIEW

File: en/docs/learn/configuring-password-reset-with-challenge.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Follow the steps below to configure WSO2 Identity Server to enable password rese
 
 4.	To mimic a forgotten password:
 
-	1.	On the Sign In screen of the WSO2 Identity Server User Portal at `https://localhost:9443/user-portal/`, click
+	1.	On the Sign In screen of the WSO2 Identity Server my account at `https://localhost:9443/myaccount`, click
```
Comment (gomathyk) [inline en/docs/learn/configuring-password-reset-with-challenge.md:103]:
```suggestion
	1.	On the Sign In screen of the WSO2 Identity Server **My Account** at `https://localhost:9443/myaccount`, click
```

File: en/docs/learn/configuring-password-reset-with-challenge.md [hunk 2]
```diff
@@ -123,4 +123,4 @@ Follow the steps below to configure WSO2 Identity Server to enable password rese
 
 		<img src="../../assets/img/learn/password-reset-form.png" alt="Password Reset form" width="500" style="border:1px solid grey">  	  
 
-	8.	Enter the user name and new password and click **Sign In**. The User Portal home screen appears. 
+	8.	Enter the user name and new password and click **Sign In**. The my account home screen appears.
```
Comment (gomathyk) [inline en/docs/learn/configuring-password-reset-with-challenge.md:126]:
```suggestion
	8.	Enter the user name and new password and click **Sign In**. The **My Account** home screen appears. 
```

File: en/docs/learn/configuring-password-reset-with-email.md [hunk 3]
```diff
@@ -131,7 +131,7 @@ Follow the steps below to configure WSO2 Identity Server to enable password rese
 
 3.	To mimic a forgotten password:
 
-	1.	Access WSO2 Identity Server User Portal at `https://localhost:9443/user-portal/`.
+	1.	Access WSO2 Identity Server my account at `https://localhost:9443/myaccount`.
```
Comment (gomathyk) [inline en/docs/learn/configuring-password-reset-with-email.md:134]:
```suggestion
	1.	Access WSO2 Identity Server **My Account** at `https://localhost:9443/myaccount`.
```

File: en/docs/learn/configuring-recaptcha-for-password-recovery.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ reCaptcha for a specific tenant.
 
 7.  You have now successfully configured reCaptcha for the password
     recovery flow. Start WSO2 Identity Server and log into the end user
-    [user portal](https://localhost:9443/user-portal).
+    [my account](https://localhost:9443/myaccount).
```
Comment (gomathyk) [inline en/docs/learn/configuring-recaptcha-for-password-recovery.md:51]:
```suggestion
    [**My Account**](https://localhost:9443/myaccount).
```

File: en/docs/learn/configuring-recaptcha-for-password-recovery.md [hunk 5]
```diff
@@ -89,7 +89,7 @@ password recovery with reCaptcha.
 
 3.  You have now successfully configured reCaptcha for the password
     recovery flow. Start WSO2 Identity Server and log into the end user
-    [user portal.](https://localhost:9443/user-portal)  
+    [my account.](https://localhost:9443/myaccount)
```
Comment (gomathyk) [inline en/docs/learn/configuring-recaptcha-for-password-recovery.md:92]:
```suggestion
    [**My Account**.](https://localhost:9443/myaccount)  
```

File: en/docs/learn/configuring-recaptcha-for-self-registration.md [hunk 6]
```diff
@@ -33,8 +33,8 @@ You can either configure the recaptcha for a tenant, or configure it globally.
     ![self-registration-enable-recaptcha](../assets/img/using-wso2-identity-server/self-registration-enable-recaptcha.png)
 7.  You have now successfully configured reCaptcha for the self
     registration flow. Start the WSO2 Identity Server and log in to the
-    user portal using the following link:
-    <https://localhost:9443/user-portal>
+    my account using the following link:
```
Comment (gomathyk) [inline en/docs/learn/configuring-recaptcha-for-self-registration.md:36]:
```suggestion
    **My Account** using the following link:
```

File: en/docs/learn/configuring-recaptcha-for-username-recovery.md [hunk 7]
```diff
@@ -45,7 +45,7 @@ reCaptcha for a specific tenant.
 
 7.  You have now successfully configured reCaptcha for the username
     recovery flow. Start the WSO2 Identity Server and log into the end
-    user [user portal](https://localhost:9443/user-portal).
+    user [my account](https://localhost:9443/myaccount).
```
Comment (gomathyk) [inline en/docs/learn/configuring-recaptcha-for-username-recovery.md:48]:
```suggestion
    user [**My Account**](https://localhost:9443/myaccount).
```

File: en/docs/learn/configuring-recaptcha-for-username-recovery.md [hunk 8]
```diff
@@ -95,7 +95,7 @@ username recovery with reCaptcha.
 
 3.  You have now successfully configured reCaptcha for the username
     recovery flow. Start WSO2 Identity Server and log into the end user
-    [user portal](https://localhost:9443/user-portal).
+    [my account](https://localhost:9443/myaccount).
```
Comment (gomathyk) [inline en/docs/learn/configuring-recaptcha-for-username-recovery.md:98]:
```suggestion
    [**My Account**](https://localhost:9443/myaccount).
```

File: en/docs/learn/configuring-totp.md [hunk 9]
```diff
@@ -92,7 +92,7 @@ and security during the authentication for this application.
 ## Configuring the user profile
 
 1.  Login to the
-    [user portal](../../learn/user-portal) and go to **Personal info**.  
+    [my account](../learn/my-account.md) and go to **Personal info**.
```
Comment (gomathyk) [inline en/docs/learn/configuring-totp.md:95]:
```suggestion
    [my account](../learn/my-account) and go to **Personal info**.  
```

File: en/docs/learn/configuring-totp.md [hunk 10]
```diff
@@ -247,9 +247,9 @@ configuration change in the `<IS_HOME>/repository/conf/deployment.toml ` file.
         property values.
 
 
-## Obtaining the QR code without using the user portal
+## Obtaining the QR code without using the my account
```
Comment (gomathyk) [inline en/docs/learn/configuring-totp.md:250]:
```suggestion
## Obtaining the QR code without using My Account
```

File: en/docs/learn/configuring-totp.md [hunk 11]
```diff
@@ -247,9 +247,9 @@ configuration change in the `<IS_HOME>/repository/conf/deployment.toml ` file.
...
 
-If you need to obtain the QR code without using the user portal,
+If you need to obtain the QR code without using the my account,
```
Comment (gomathyk) [inline en/docs/learn/configuring-totp.md:252]:
```suggestion
If you need to obtain the QR code without using **My Account**,
```

**Review by gomathyk — CHANGES_REQUESTED** [review]

**Comment by gomathyk** [comment]
1. Please change the formatting from my account to **My Account** wherever applicable.
2. Please use the relative path for internal links. (Example - ../learn/my-account instead of ../learn/my-account.md

File: en/docs/learn/configuring-totp.md (hunk 9 above)
Comment (Sahilgandas1) [reply en/docs/learn/configuring-totp.md:95]: ../learn/my-account is not working i have checked but ../learn/my-account.md works

**Comment by Sahilgandas1** [comment]
@gomathyk i have done formatting in commit no 823048e you can check it

**Comment by somindatommy** [comment]
@gomathyk can you revieew this?

**Comment by gomathyk** [comment]
Thank you for contributing, @Sahilgandas1 !

**Review by gomathyk — APPROVED** [review]

**Pull request merged**